### ISSUE TITLE
Getting a Kernel panic when trying to use it with EzTime library

### ISSUE THREAD

**Issue by witnessmenow** [toblum/TetrisAnimation#3]
More info to follow

**Comment by tablatronix** [comment]
I have traced these panics to using a timer or ISR and doing too much inside it.
Post anything different if you see it

**Comment by witnessmenow** [comment]
Will try out your changes on the Matrix panel when I get a change

**Comment by ROSW6341** [comment]
I do not know the underlying cause but I was able to circumvent the problem by copying core_esp_main.cpp from
C:\Users\<USER>\AppData\Local\arduino15\packages\esp8266\hardware\esp8266\2.5.2\cores\esp8266 
into the sketch directory then commenting out the 'panic()' on line 103 of the copied file.

**Comment by tablatronix** [comment]
Yeah its strange because it is specifically calling panic exception because of the 'in interrupt' check, not sure what the other 2 checks are. Its odd

afaik , this is not in an interrupt

**Comment by witnessmenow** [comment]
Someone has a workaround for this on the WiFi Tetris clock repo

https://github.com/witnessmenow/WiFi-Tetris-Clock/issues/8

**Comment by ROSW6341** [comment]
Worked for me and much cleaner than the core_esp_main.cpp hack I posted above

**Comment by tablatronix** [comment]
That is what I did also, but it did still wdt now and then during long draws, but it is probably the easiest fix. Of course you can just not use a timer or use a timer library that just uses timekeeping and a process()

**Comment by tablatronix** [comment]
Is it possible to compile with `no-exceptions` build flag and bypass this also ?

**Comment by tablatronix** [comment]
aha

https://github.com/esp8266/Arduino/issues/6283
